releasenotes/notes/ecs-fargate-collect-net-stat-metrics-22ec649320df16b4.yaml

```diff
@@ -0,0 +1,3 @@
+enhancements:
+  - |
+    Collect net stats metrics (RX/TX) for ECS Fargate containers.
```

Comment:
nit: should we clarify that we're referring to the live containers page ?

Reply:
Goog point, thanks @ahmed-mez, adding.